coldfront/plugins/qumulo/utils/active_directory_api.py

```diff
@@ -41,6 +41,18 @@ def get_user(self, wustlkey: str):
 
         return self.conn.response[0]
 
+    def get_member(self, account_name: str):
+        self.conn.search(
+            "dc=accounts,dc=ad,dc=wustl,dc=edu",
+            f"(&(|(objectclass=group)(objectclass=person))(sAMAccountName={account_name}))",
+            attributes=["sAMAccountName", "objectclass"],
+        )
+
+        if not self.conn.response:
+            raise ValueError("Invalid account_name")
```

Comment:
Is it technically an invalid `account` (!) name?

Reply:
I'm not sure...not sure what else to classify it as, but the most relevant Domain Component (dc) in the AD search query is `accounts`.  Open to suggestions, though

Reply:
elsewise it can be SAMLname or whatever it is called, but that might feel less helpful/more obscure